pki.spec

```diff
@@ -231,19 +239,20 @@ BuildRequires:    mvn(org.jboss.resteasy:resteasy-jaxrs)
 BuildRequires:    mvn(org.jboss.resteasy:resteasy-client)
 BuildRequires:    mvn(org.jboss.resteasy:resteasy-jackson2-provider)
 BuildRequires:    mvn(org.jboss.resteasy:resteasy-servlet-initializer)
+
 %endif
 
-BuildRequires:    mvn(org.apache.tomcat:tomcat-catalina) >= 9.0.62
-BuildRequires:    mvn(org.apache.tomcat:tomcat-servlet-api) >= 9.0.62
-BuildRequires:    mvn(org.apache.tomcat:tomcat-jaspic-api) >= 9.0.62
-BuildRequires:    mvn(org.apache.tomcat:tomcat-util-scan) >= 9.0.62
+BuildRequires:    mvn(org.apache.tomcat:tomcat-catalina) >= 10.1.24
+BuildRequires:    mvn(org.apache.tomcat:tomcat-servlet-api) >= 10.1.24
+BuildRequires:    mvn(org.apache.tomcat:tomcat-jaspic-api) >= 10.1.22
+BuildRequires:    mvn(org.apache.tomcat:tomcat-util-scan) >= 10.0.62
```

Comment:
Should this be Tomcat 10.1.33 as in `pom.xml`?

Reply:
Will do.

Reply:
Updated in last push.